values.yaml

```diff
@@ -107,75 +106,29 @@ api:
   # 'existingSecret' specifies name of externally created secret
   # existingSecret:
 
-  initMigrate:
-    enabled: true
-    backoffLimit: 6
-  migrate:
-    enabled: true
-
-keyManager:
-  enabled: true
-
-  replicaCount: 1
-
-  service:
-    type: ClusterIP
-    http:
-      port: 8081
-    metrics:
-      port: 8082
-
-  serviceAccount:
-    # Specifies whether a service account should be created
-    create: true
-    # Annotations to add to the service account
-    annotations: {}
-    # The name of the service account to use.
-    # If not set and create is true, a name is generated using the fullname template
-    name: ""
-
-  podAnnotations: {}
-
-  podSecurityContext: {}
-  # fsGroup: 2000
-
-  securityContext: {}
-    # capabilities:
-    #   drop:
-    #   - ALL
-    # readOnlyRootFilesystem: true
-    # runAsNonRoot: true
-    # runAsUser: 1000
-
-  resources: {}
-  #  limits:
-  #  cpu: 300m
-  #  memory: 128Mi
-  # requests:
-  #  cpu: 100m
-  #  memory: 32Mi
-
-  nodeSelector: {}
-
-  tolerations: []
-
-  affinity: {}
+  auth:
+    jwt:
+      issuerUrl: https://consensys.eu.auth0.com
+      audience: https://orchestrate.consensys.net
+      claims: https://api.orchestrate.network
 
+migrate:
+  backoffLimit: 6
   environment: {}
   environmentSecrets: {}
   # 'existingSecret' specifies name of externally created secret
   # existingSecret:
 
-  migrate: false
 
-  vaultAgent:
-    enabled: false
-    role: "client"
-    image:
-      repository: vault
-      tag: 1.6.2
-    config:
-      wrapTTL: ""
+qkm:
+  enabled: true
+  proto: https
+  fullname: "quorumkeymanager"
+  url: ""
+  port: 8080
+  orchestrate:
+    storeName: "orchestrate-eth"
```

Comment:
I cannot find any usage of this value in the helm template, could you tell me where to look at? thxs